WBooks/Screens/BookDetails/BookDetailsViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  BookDetailsViewController.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 05/04/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class BookDetailsViewController: BaseViewController {
+    private let _bookDetailsViewModel: BookDetailsViewModel
+    private let _view: BookDetailsView = BookDetailsView.loadFromNib()!
+    
+    // MARK: - UIViewController
+
+    required public init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        fatalError("init(nibName:bundle:) has not been implemented")
+    }
+    
+    // MARK: - BookDetailsViewController
+
+    init(viewModel: BookDetailsViewModel) {
+        _bookDetailsViewModel = viewModel
+        super.init(nibName: .none, bundle: .none)
+    }
+    
+    override public func loadView() {
+        view = _view
+        _view.bookTitle.text = _bookDetailsViewModel.bookModel.title
+        _view.bookGen.text = _bookDetailsViewModel.bookModel.genre
+        _view.bookYear.text = _bookDetailsViewModel.bookModel.year
+        _view.bookAuthor.text = _bookDetailsViewModel.bookModel.author
+        _view.bookStatus.text = _bookDetailsViewModel.bookModel.status
+        _view.updateStyles(newStatus: _bookDetailsViewModel.bookModel.status)
+        if _bookDetailsViewModel.bookModel.image != nil {
+            _view.bookImage.loadUrl(from: _bookDetailsViewModel.bookModel.image ?? "")
```

Comment:
In bookDetailsViewModel add a property to get this value and return nil if it's ""

Reply:
Remember that the view/controller should not know anything about the model